### DISCUSSION TITLE
Unable to create a pull request in maya-usd

### DISCUSSION THREAD

**Discussion by NickWu** [Autodesk/maya-usd#3577] [General] [answered]
Hi, I am not sure if it's related to maya-usd's repo setting but I couldn't create a pull request in maya-usd with the error message **"Pull request creation failed. Validation failed: must be a collaborator"**. But I could create one in the past. Example: https://github.com/Autodesk/maya-usd/pull/2927. I wonder what's changed?

**Comment by seando-adsk** [comment]
Hi Nick, we had to make some modifications recently because of some updated security policies at Autodesk. I changed all the `outside collaborators` to read access (instead of write). I couldn't find you in the user list anymore, so I must have removed you. But even if I hadn't removed you I would have changed your status to "Read".

That this means now is that all outside contributors need to submit the PR from a fork instead of directly in the github repo.

Sean

**Reply by NickWu** [reply]
Thanks for the reply Sean. You meant currently regardless of being in the user list or not, I'd ned to submit a PR from a fork? Is that the current policy for outside contributors?

**Reply by seando-adsk** [reply]
Yes. We are not allowed to add anyone who is not part of the Autodesk organization.

**Reply by NickWu** [reply]
Hi Sean, in the last failed try, I was trying to create a PR from Animal Logic's `maya-usd` fork, but it still didn't work where it used to work in the past. 

<img width="1703" alt="Screenshot 2024-01-30 at 10 54 28" src="https://github.com/Autodesk/maya-usd/assets/935278/eb479d2f-0766-4078-90eb-7835b148d9b0">

Is it also related to the current maya-usd's security policy?

**Reply by seando-adsk** [reply]
Sorry I have never seen that error before. I looked at the AnimalLogic fork and compared it to some other forks where users have been able to create pull-requests. The only difference I can see is that the AL fork doesn't have the `dev` branch. But I don't think that is the reason your PR failed, because I was able to create a [PR 3587](https://github.com/Autodesk/maya-usd/pull/3587) (which I closed) from your branch on the AL fork. Were you trying to create the pull request from the AL Github fork, ie here:
https://github.com/Autodesk/maya-usd/compare/dev...AnimalLogic:maya-usd:NickWu/save_custom_variant_fallbacks

Sean

**Reply by NickWu** [reply]
Yes, that is the branch https://github.com/Autodesk/maya-usd/compare/dev...AnimalLogic:maya-usd:NickWu/save_custom_variant_fallbacks. Just tried again to create a PR but no luck with the same error message. I am sure I could create one last year.

**Reply by seando-adsk** [reply]
Would it be possible for you to create a new fork of maya-usd in your own github space and see if you are able to create a pull-request from there? 

Sean

**Reply by NickWu** [reply]
It looks like I am able to create a PR from my personal fork to maya-usd. https://github.com/Autodesk/maya-usd/pull/3588

**Reply by NickWu** [reply]
We are investigating to see if there is something wrong in our Animal Logic's fork.

**Reply by NickWu** [reply, marked as answer]
Ok, we have found the cause and it's resolved. I am able to create PRs now. Thanks for the help.
